Add release notes for 7.6 (#822)
chef/cookstyle: RELEASE_NOTES.md

## Cookstyle 7.6

## RuboCop 1.9.0

> The RuboCop engine that powers Cookstyle has been updated to 1.7.0. This new release improves autocorrection and resolves a large number of detection errors.

Reviewer: ```suggestion
Updated RuboCop--which is the engine behind Cookstyle--to 1.7.0. The new release improves autocorrection and resolves a large number of detection errors.
```
